src/data/configs/cc_high_quality_20bn.yaml

```diff
@@ -0,0 +1,38 @@
+sources:
+  - name: cc_en_middle
+    source_coefficient: 0  # Remove all middle quality CC
+  - name: falcon
```

Comment:
For the high quality can we remove Falcon and MegaWika also? Falcon is of nebulous quality (I've found it has a lot of duplicates) and as an author of MegaWika it's quality can be pretty mixed -- depends on how AI2 filtered it.

Reply:
I didn't know about MegaWika! But 100% agreed for C4 and especially Falcon.

Reply:
Done